charts/kubedbcom-postgres-editor-options/templates/db.yaml

```diff
@@ -21,7 +22,11 @@ spec:
   authSecret:
     name: {{ .Values.spec.authSecret.name }}
 {{- end }}
+{{- if ge $pgv 10 }}
```

Comment:
It should be 11. The support for scram is from 11.

Reply:
Fixed